### PR TITLE
Update documentation

### LINKED ISSUE #10902: `Cart Line Items » saleBadgePriceFormat`: Add code snippet and screenshot
Recently, we've updated the "Available Extensibility Interfaces for the Cart and Checkout blocks" post. In this post, we introduced new code snippets and screenshots. This issue aims to add the following code snippet and screenshot to the [`Cart Line Items » saleBadgePriceFormat`](https://github.com/woocommerce/woocommerce-blocks/blob/6083484477410fd99d1efd5ca909288eb4d19963/docs/third-party-developers/extensibility/checkout-block/available-filters.md?plain=1#L26) section:

```js
const { registerCheckoutFilters } = window.wc.blocksCheckout;
 
// Adjust sale badge price format of the cart line items.
registerCheckoutFilters( 'example-extension', {
  saleBadgePriceFormat: ( value, extensions, args ) => {
    // Return early since this filter is not being applied in the Cart context.
    // We must return the original value we received here.
    if ( args?.context !== 'cart' ) {
      return value;
    }
    return '<price/> per item';
  }
} );
```

screenshot-2023-08-24-at-13 13 35

---

> **Note**
> The code snippet and screenshot should be placed below the table with the filter names and the sentence _"Each of these filters has the following arguments [...]"_, and should have the following subheading
```md
### saleBadgePriceFormat
```

### PR DESCRIPTION
**Pull request by corners2wall**
## What

Closes #10902 

Fixes #

## Testing Instructions

_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1.
2.
3.

* [x] Do not include in the Testing Notes 
* [ ] Should be tested by the development team exclusively 

## Screenshots or screencast

Before:
Screenshot from 2023-09-13 15-11-52

After:
Screenshot from 2023-09-13 15-09-06

## WooCommerce Visibility

Required:

* [ ] WooCommerce Core
* [ ] Feature plugin
* [ ] Experimental
* [x] N/A

## Checklist

Required:
* [x] This PR has either a `[type]` label or a `[skip-changelog]` label.
* [ ] This PR is assigned to a milestone.

Conditional:
* [ ] This PR has a changelog description (if `[skip-changelog]` label is not present).
* [ ] This PR adds/removes a feature flag & I've updated [this doc](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/internal-developers/blocks/feature-flags-and-experimental-interfaces.md).
* [ ] This PR adds/removes an experimental interfaces, and I've updated [this doc](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/internal-developers/blocks/feature-flags-and-experimental-interfaces.md).
* [ ] This PR has been accessibility tested.
* [x] This PR has had any necessary documentation added/updated.

## Changelog

> Add suggested changelog entry here.

### CODE REVIEW

**Review by nielslange — CHANGES_REQUESTED** [review]
Thank you so much for your contribution, @corners2wall. We appreciate this very much. 

While reviewing this PR, I noticed a few things, that should be changed:
- Please replace _"Update documentation"_ with a speaking title, such as _"Cart Line Items » saleBadgePriceFormat: Add code snippet and screenshot"_, which is the title of the corresponding issue.
- Please provide information in the section _"What"_ so that a reviewer can see what had been changed. (Example: #10930)
- Please provide information in the section _"Why"_ so that a reviewer can see why this change was needed. (Example: #10930)
- Please provide information in the section _"Testing Instructions"_ so that a reviewer can see how to test this PR. (Example: #10930)
- The section _"Screenshots or screencast"_ shows the initial screenshot as _"Before"_ and the code example and the new screenshot as _"After"_. This indicates that the original screenshot had been removed, which is not the case. As the section _"Screenshots or screencast"_ is not really needed for this PR, you can remove it.
- Currently, the code example and the new screenshot are placed directly below the title _"Cart Line Items"_. Please move both the code example and the new screenshot below the table with the filter names and the sentence _"Each of these filters has the following arguments [...]"_, and add the following subheading:
```md
saleBadgePriceFormat
```
I haven't mentioned this in the initial issue, as I noticed this too late. Sorry for that.

---

Other than that, the PR looks good. If you have addressed these points, we should be able to approve and merge this PR. 

If you have any questions regarding the points I mentioned, please let me know, and thanks again for your contribution. 🙏

File: docs/third-party-developers/extensibility/checkout-block/available-filters.md [hunk 1]
```diff
@@ -2,29 +2,46 @@
...
+ 
+// Adjust sale badge price format of the cart line items.
+registerCheckoutFilters( 'example-extension', {
+  saleBadgePriceFormat: ( value, extensions, args ) => {
+    // Return early since this filter is not being applied in the Cart context.
+    // We must return the original value we received here.
+    if ( args?.context !== 'cart' ) {
+      return value;
+    }
+    return ' per item';
```
Comment (nielslange) [inline docs/third-party-developers/extensibility/checkout-block/available-filters.md]:
```suggestion
    return '<price/> per item';
```

Comment (nielslange) [reply docs/third-party-developers/extensibility/checkout-block/available-filters.md]: For some reason, the part `<price/>` got lost when pasting the code example. I just added it to the issue. Please also add it to the PR. 😀

**Comment by nielslange** [comment]
Hello @corners2wall, I see that you committed my suggestions. Could you also address the points that I mentioned in https://github.com/woocommerce/woocommerce-blocks/pull/10935#pullrequestreview-1625829820? 😀

**Comment by corners2wall** [comment]
@nielslange No, I don't want to understand this, sorry

**Comment by nielslange** [comment]
> @nielslange No, I don't want to understand this, sorry

Thanks for letting me know. In https://github.com/woocommerce/woocommerce-blocks/pull/10935#pullrequestreview-1625829820, I am mainly talking about the description of this PR and not the code itself. 

Let's compare https://github.com/woocommerce/woocommerce-blocks/pull/10976 with this PR:

Screenshot 2023-09-16 at 11 22 52

If you don't understand my suggestions because of a language barrier, maybe it would help if you use [Google Translate](https://translate.google.com/) to translate my comment into your language. 

Please let me know if that helps or if I shall explain my suggestions in detail.

**Comment by nielslange** [comment]
Hello @corners2wall, I haven't heard back from you. Based on your comment in https://github.com/woocommerce/woocommerce-blocks/pull/10935#issuecomment-1721533316, I assume you no longer want to work on this issue. I'm going to close this PR for now. That said, I'd be happy to see you again as a contributor to WooCommerce Blocks in the future.

**Pull request closed without merging**